tools/reviewer/web/src/app/dashboard/reviews-panel/file-changes/file-changes.service.ts

```diff
@@ -16,17 +16,18 @@ interface AddCommentResponse {
 // The service receives data from one place of
 // diff component (and children) and send to another.
 @Injectable()
-export class DiffService {
+export class FileChangesService {
   // A number of the line, where user cursor is hovering right now.
   private hoveredLine: number;
-  // Is cursor hovering above new/old code?
+  // Is cursor hovering above new code?
   private isHoveredNewCode: boolean;
 
   // Subjects, which send data:
-  lineHeightChanges: Subject<HeightResponse> = new Subject<HeightResponse>();
-  openCommentsChanges: Subject<number> = new Subject<number>();
-  closeCommentsChanges: Subject<number> = new Subject<number>();
-  newComment: Subject<AddCommentResponse> = new Subject<AddCommentResponse>();
+  lineHeightChanges = new Subject<HeightResponse>();
```

Comment:
Are we not using strongly typed here?

Reply:
We need to make decision for that situations too.
I think it's excessively with `new`.
Variable has type so you have full IDE type check functionality and it's pretty obvious for reviewer, which type it's.
I don't know, maybe we should use strongly typed way here too. But I prefer not.